<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40,43,48);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxMid">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>301</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_Start">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>460</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 153, 117);
color: #FFFFFF;
font: 75 12pt &quot;Microsoft YaHei&quot;;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Iniciar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_titulo_Process">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 153, 117);
font: 87 14pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>Procesos</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_ProcessContainer">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>211</width>
       <height>331</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGraphicsView" name="graphicsView_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_13">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_14">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_15">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_16">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_17">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_18">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_19">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_20">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_21">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_22">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_23">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>250</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_24">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_25">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>11</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>12</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>13</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>14</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>15</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>16</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>17</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>18</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>19</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>20</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>21</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>22</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>290</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>23</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>290</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>24</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>25</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFirst">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="imageProductorAwake">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/ConsumerAwake/farmer (1).png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/newPrefix/farmer.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="textBox_StatusProductor">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>161</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 62, 64);
color: rgb(255, 255, 255);
border: 1px solid rgb(0, 153, 117);
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_Productor">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 153, 117);
font: 87 14pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>Productor</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxLast">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>221</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_tittleConsumer">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 153, 117);
font: 87 14pt &quot;Segoe UI Black&quot;;
</string>
     </property>
     <property name="text">
      <string>Consumidor</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textBox_StatusConsumer">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>161</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 62, 64);
color: rgb(255, 255, 255);
border: 1px solid rgb(0, 153, 117);
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="consumer.qrc"/>
 </resources>
 <connections/>
</ui>
